CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.3.4] - 2023-07-21
+
+### Added
+
+Added an `ordered_jobs` list as an attribute to the `Batch` object in which jobs are ordered by creation date.
```

Comment:
Aren't Python dicts ordered in newer pythons (after 3.6 or something like that)?
So we could just keep the `jobs`? or does the db now return a list?

Reply:
The db return a list of Jobs